clearnode/docs/API.md

```diff
@@ -1128,26 +1128,34 @@ In the request, the user must specify funds destination. After the channel is cl
 
 ### Resize Channel
 
-Adjusts the capacity of a channel.
+Adjusts the capacity of a channel by depositing or withdrawing funds.
+
+**IMPORTANT**: Once a resize is requested, the channel enters a "resizing" state. The resize state must be submitted to the blockchain to complete the resize operation. If the resize transaction is not submitted, the channel will remain in the resizing state and the only way to recover is to close the channel. Then user can reopen a channel.
+
+**Fund Locking**: When withdrawing funds (negative `resize_amount`), the funds being withdrawn are immediately locked until the resize transaction on-chain confirmation is received.
 
 **Request:**
 
 ```json
 {
   "req": [1, "resize_channel", {
     "channel_id": "0x4567890123abcdef...",
-    "allocate_amount": "200000000",
-    "resize_amount": "1000000000",
+    "resize_amount": "-1000000000",
+    "allocate_amount": "+1000000000",
     "funds_destination": "0x1234567890abcdef..."
   }, 1619123456789],
   "sig": ["0x9876fedcba..."]
 }
 ```
 
-`allocate_amount` is how much more token user wants to allocate to this token-network specific channel from his unified balance.
-`resize_amount` is how much user wants to deposit or withdraw from a token-network specific channel.
+**Request Parameters:**
+
+- `channel_id`: The ID of the channel to resize
+- `resize_amount`: Amount to deposit (positive) or withdraw (negative) from the channel.
+- `allocate_amount`: Amount to allocate (positive) or deallocate (negative) to this specific channel from the user's unified balance.
+- `funds_destination`: an address where funds should be sent (used for withdrawals)
```

Comment:
By the way, the fact that this is a required parameter is a confusing when performing a deposit. I think it should be made optional.